### PR TITLE
need fix mean_delta_cpu?

### PR DESCRIPTION
**Pull request by automateljw**
This function is missing the latter part of the formula
paper link: https://arxiv.org/abs/1502.03167
paper name: Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift
mean_delta rely on variance_delta

### CODE REVIEW

**Comment by lixiaohui2020** [comment]
I have found this problem and think it is wrong. @pjreddie

**Comment by pjreddie** [comment]
The formula in the paper is, as they state, before simplification. The sum of differences between values and means evaluates to zero hence you can drop the portion of the term you are referring to.
